hw03_adkinsh2.py

```diff
@@ -0,0 +1,73 @@
+#################################################################################
+# Author: Harry Adkins
+# Username: adkinsh2
+#
+# Assignment: HW03
+# Purpose: Drawing stars
+# Google Doc Link: https://docs.google.com/document/d/1DqjcrDnNRH_vvrzYEIaPAd95IWsKW53YRXWLzz5ADKw/edit
+#
+#################################################################################
+# Acknowledgements: Python documentation
+#
+#
+#################################################################################
+
+import turtle
+
+def drawStar():
+    """
+    Draws six 5-pointed stars
+    """
+    bigStar = turtle.Turtle()
+    bigStar.pensize(4)
+    bigStar.color("white")
+
+    bigStar.penup()
+    bigStar.left(60)
+    bigStar.forward(200)
+    bigStar.right(60)
+    bigStar.pendown()
+
+    for i in range(6):
+        for j in range(5):
+            bigStar.forward(30)
+            bigStar.right(144)
+        bigStar.penup()
+        bigStar.right(60)
+        bigStar.forward(100)
+        bigStar.pendown()
+
+def drawMoon():
+    """
+    Draws a crescent moon
+    """
+    moon = turtle.Turtle()
```

Comment:
You don't need to fix this, but I wanted to make a note of it. 

It looks like you chose to spawn a new turtle inside the functions. You likely did this because of scoping issues; the turtle you created in main() wasn't accessible inside your function. 

Another way you could have handled this is passing the turtle into the function when it was called:

```
def drawMoon(t):
    ...

def main():
    ...
    t = turtle.Turtle()
    drawMoon(t)
    ...
```